<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COMP3004 Final Project - Team 78</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>541</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGraphicsView" name="displayScreen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>431</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="upButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>190</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>↑</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>240</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>↓</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>430</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>430</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PAUSE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>430</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menuButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>MENU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>POWER</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="contactLight">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="treatmentLight">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QGraphicsView" name="contactLostLight">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="option1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="option2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="option3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="selectButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>280</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>SELECT</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="battery">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>450</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>430</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Battery</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateInput">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>350</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2024</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>431</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="treatmentLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>311</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Treatment</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QCustomPlot" name="plotWidget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>411</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="batteryCharge">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>460</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 120, 0)</string>
     </property>
     <property name="text">
      <string>Charge</string>
     </property>
    </widget>
    <widget class="QPushButton" name="contactButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>5</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(237, 51, 59)</string>
     </property>
     <property name="text">
      <string>Contact</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>675</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
